.github/workflows/test.yaml

```diff
@@ -145,6 +153,14 @@ jobs:
             # Distributed trial is not yet suported on Windows.
             trial-args: '--reactor=iocp'
 
+          # Windows, newest Python supported version with iocp reactor.
```

Comment:
Please see the comment on line 148

The comment is no longer valid, as 3.10 is no longer the newest Python.

The idea is that instead of adding a new job for 3.11, we will just update the "newest" job 

So my suggestion is to delete this job, and at line 149, replace 3.10 with 3.11

makes sense?

Reply:
I'm confused, do we want to update this so the "newest" jobs run on 3.11 and make 3.11 the default python?  I had this previously, but was asked to revert it.

Reply:
Sorry for the confusion.

I know that at some point you have updated 3.11 as the default Python.
This means that our Sphinx and pydoctor jobs and other non-test releated jobs were exectued on 3.11... and some were failing.
So my suggestion was to go with baby steps, and only use 3.11 for the python tests.

Does it make sense?

Reply:
Yes, that makes sense. That should be updated w/ my latest commit.